README.md

```diff
@@ -12,3 +12,35 @@ The following plugins are offered:
 	Allows to package and distribute Python modules during Maven builds
 
 
+## Usage
+
+
+To use the plugin, add the following in your `pom.xml`
+
+```xml
+
+<build>
+        <plugins>
+        ...
+            <plugin>
+                <groupId>io.sqooba</groupId>
+                <artifactId>maven-python-plugin</artifactId>
+                <version>1.1.2</version>
```

Comment:
This needs to be version `1.1.1.` as per your pom below

Reply:
Removed in 4ec80f06565ef4df7bbde1310116f925e03b8ca8